deploy/playbooks/deploy_vagrant.yml

```diff
@@ -17,3 +17,4 @@
      repos_root: "/opt/repos"
      branch: "master"
      development_server: true
+     app_user: "nginx"
```

Comment:
I've tested this branch in a local VM :)

I do think we'll need to run some `chown -R nginx:nginx` commands as cleanup if we make this change in production, though, since there is existing data there that is already owned by "admin". Something like this should work:

```
chown -R nginx:nginx /opt/chacra/log /opt/binaries /opt/repos /var/log/circus/
```